.github/workflows/ci.yml

```diff
@@ -195,8 +195,9 @@ jobs:
         with:
           toolchain: stable
       - name: Build bevy
+        # this uses the same command as when running the example to ensure build is reused
         run: |
-          cargo build --features "bevy_ci_testing,trace,trace_chrome"
+          TRACE_CHROME=trace-alien_cake_addict.json CI_TESTING_CONFIG=./github/example-run/alien_cake_addict.ron cargo build --example alien_cake_addict --features "bevy_ci_testing,trace,trace_chrome"
```

Comment:
Hmm should this be .github/example-run/…?

Reply:
nop

Reply:
yes